File: Logitude/AngularModules/AngularModules/Customs/EntityPMs/Extended/GeneralDataView.ts
 import { AmendmentView, error  } from './AmendmentView';
export class GeneralDataView {
    CorrectionDate: Date;
    Version: string;
    AdditionalInformation: AdditionalInformationView[];
    AmendmentViews: AmendmentView[];
    SystemMessageViews: any;//error[];// AOT Compile error Cannot find name 'error'
    References: ReferenceView[];
}
export class AdditionalInformationView {
    StatmentName: string;
    Content: string;
}


export class ReferenceView {
    ReferenceTypeName: string;
    RefernceStatusName: string;
    RefernceID: string;
    Remarks: string;
    RefernceInputTypeName: string;
}

File: Logitude/Logitude.Customs.BL/Messaging/LogitudeClient/DeclarationCorrection/GeneralDataView.cs
using Logitude.Customs.BL.Messaging.LogitudeClient.DeclarationErrorPointer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Logitude.Customs.BL.Messaging.LogitudeClient.DeclarationCorrection
{
   public class GeneralDataView
    {
       public DateTime? CorrectionDate { get; set; }
       public string Version { get; set; }

       public List<AdditionalInformationView> AdditionalInformation { get; set; }
       public List<AmendmentView> AmendmentViews { get; set; }
        public List<error> SystemMessageViews { get; set; }
    }
}
